var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -360,8 +360,8 @@ def is_disabled(text):
         return results
 
     def setup_build_environment(self, env):
-        env.unset("F90")
         env.unset("F90FLAGS")
+        env.unset("F90")
```

Comment:
Why would the order of these statements matter?

Reply:
Or is this a PR just for testing?

Reply:
This was to check if the error in `develop` was a false positive, by changing the hash and triggering a rebuild in a PR. See https://spackpm.slack.com/archives/C02N33GM28H/p1664280616596179